### DISCUSSION TITLE
論研究所的必要性

### DISCUSSION THREAD

**Discussion by Yanxxx7979** [1101-GS4719-C/Python-Community-GS4719-C#1724] [General]
這堂課的大部分同學都來自於理工科系
而大家好像都有種共識就是未來會繼續升學進入研究所
然而最近我在思考,所謂申請研究所的真正動機
我自己列了一些可能因素
1.真心喜愛這們學科或是當研究領域
2.為了薪水(碩士與學士的薪水差距及CP值)
3.大家都要讀我也要讀，跟緊梯隊
4.逃避出社會...
我自己個人是以上四點都有權重啦，但好奇大家對於研究所和生涯規劃的看法

**Comment by THH1118** [comment]
自己的話是比較偏向選擇三的理由，當然其實身邊也有看過沒有繼續讀下去的人，但的確是極少數，可能也是因為沒有很明確的規劃也勇氣選不一樣的道路吧，今年的推甄也已經告一段落了，其實對自己研究所的規劃還挺茫然的（就雖然有自己想做的東西但也不太知道自己真的開始讀了會是怎麼樣的生活這樣）

**Comment by 12Jesse24** [comment]
我會選2.畢竟起薪也是有差，明明知道讀了薪水比較高何樂而不為呢，但其實進公司久了，多半還是看你的能力給你對應的薪水的，也有聽說沒讀研究所的人最後薪水贏超多讀碩博士的，所以自己有能力就不怕薪水給的少，我自己是這樣認為的拉。

**Comment by uwje** [comment]
我認為大部分人都是二或三吧，現在真的因為想要研究而去讀研究所的人已經不多了，大多是因為社會氣氛，認為碩士學歷比較好，或公司開給碩士的底薪也較高。

**Comment by eric202501** [comment]
以我自己的科系來說應該是2，大學畢業直接去工作的人通常都是很有技術力或是在特定領域已經有成就的人。不然大部分還是會選擇碩士，能提高自己未來的期望值

**Comment by 111426027** [comment]
2 其實自己並不是很喜歡讀書，但是希望未來的薪水比較高，所以才讀研究所。

**Comment by sorn32** [comment]
主要是2，我是跨考想要換掉學歷的title，所以才決定要讀研(｡A｡)

**Comment by v0ouo** [comment]
我自己會選擇2.，主要是我們科系大學去工作跟研究所出去工作得到的薪水真的差很多，而且上研究所也可以學到不少東西，對未來出社會也比較有幫助。

**Comment by davidteng00000** [comment]
選擇進入研究所往往涉及多重因素。除了對學科熱愛、薪酬與學歷相關外，追求知識深度和學術成就也是動機之一。個人目標、興趣和職涯規劃在做出決定時扮演重要角色，影響著未來的發展與專業道路。

**Comment by Patsy17** [comment]
2 為了未來的薪水，只有提高自己的學歷才有籌碼去跟公司談到更高的薪水
我朋友理工學士畢業，工作一年，後來有個碩畢的起薪直接比他現在的薪水還高

**Comment by mrweixd** [comment]
趁有能力就多讀幾年，老了學習能力會下降，會學得更痛苦

**Comment by alvin119** [comment]
我會選1跟2，
選1是因為自己的科系的緣故，如果沒讀研究所感覺會少了一個自己專精的領域，將來選擇工作等於少了一些選擇。
選2的原因就是因為碩士當然起薪大部分比學士高。

**Comment by chuanmin0317** [comment]
我個人會想讀研究所的原因可能會比較偏向2吧，在台灣的話理工科系有沒有讀研究所感覺起薪是跟大學學歷差距蠻大的，所以就會希望以研究所為目標

**Comment by FakeWood** [comment]
我自己是2、3，還是俗套的屈服於現實，一切都是為了錢、為了生活，
我想，如果不用為錢煩惱的話，我只會整天遊山玩水或是躺在家裡書耍廢吧，
感覺有點糟糕，不過反正我也沒那麼有錢，講講罷了

**Comment by YANRONG0123** [comment]
我會選二，畢竟有機會的話，能夠多贏別人一個條件，也可以順便模擬之後上班會做到累死的狀況，而且蠻多企業門檻都要碩士以上，此外，讀研究所還可以學到大學學科以外的專長，在選擇產業上，能夠比別人還多。

**Comment by YifenLin** [comment]
我自己比較偏向2，但是我想薪水跟學歷成正比這件事肯定有它的原因，只是真正的原因是甚麼可能要等到我真的讀過研究所才會知道。

**Comment by willychiuchiu** [comment]
我是屬於2、3比較多，尤其我所在的這個系，是否有研究所學歷對未來工作的薪水就會有很明顯的差距，而且身邊的同學們很多都從大二就開始補習準備研究所，自己也會開始緊張，於是也跟著報補習班了⋯

**Comment by 20050201Shawn** [comment]
我覺得現階段讀研究所對大部分理工科的人來說是利大於弊，好像就單純沒有不去讀的理由這樣，如果還沒對未來沒有明確想法也可以先讀看看再說。

**Comment by wanka2030** [comment]
讀研究所的趨勢好像都是屬於某一派人的想法，像是理工系如果沒有研究所的加持的話，大多數應該會處於渾渾噩噩的情況下就被丟入社會之中，而研究所雖然時間很短暫，但卻是能夠讓學生看清楚自己到底想要甚麼的詩間。

**Comment by tzurou** [comment]
我自己想選擇讀研究所是因為薪水，如果多努力兩年能讓我獲得比別人多一點的薪水的話，我會願意付出這些時間，當然也還有父母對自己的期許，家裡支持的話應該會是考研究所的動力，加上我自己也還沒準備好出社會(哭

**Comment by LinChloe** [comment]
2
我想賺很多很多的錢錢，不能讓研究所阻擋我賺錢的道路。
但也有一部分是對未來也有點害怕，想去研究所被論文鞭策一下，再去面對更殘酷的世界

**Comment by happy246810** [comment]
2跟3
因為大家都在說是否有讀研究所的薪水差了十萬八千里，同時周遭的人也都在說以後要上哪一所大學的研究所，所以才會有想上研究所的想法。
而關於第四點，我覺得多少有一點，畢竟你出社會前都是在學校度過，而研究所就可以當成是一個過渡期，適應一下。

**Comment by yyujimmy** [comment]
真心喜愛學科或研究領域：這是最重要的動機之一。對於研究所的學習和研究有濃厚的興趣和熱情，願意花時間和精力深入探索相關領域。
求取薪水或職業發展：碩士學位可能會為個人的職業發展帶來更多的機會和挑戰。一些職業可能需要碩士以上的學歷，並且碩士學位通常會提高就業的競爭力和薪資待遇。
跟隨梯隊或壓力：有時候，周圍的同學或社會環境可能會對個人的決定產生影響，感覺需要跟上同儕的步伐或壓力，進入研究所也許是其中之一。
逃避出社會：有些人可能會選擇繼續深造，以避免直接進入職場的壓力和挑戰，也許他們認為研究所生活更自由或者對於未來的職業規劃尚未確定。

**Comment by KH1126** [comment]
我未來會考慮讀研究所。
讀到目前大三，在修過較多選修課以後，才找到自己喜歡電機系哪個領域。
因此為了更專精，會在研究所學習更多專業知識，幫助為未來銜接職場。

**Comment by jeff110303549** [comment]
大家對於申請研究所的動機可能因人而異，一些共通的觀點和考量因素有像職業發展和薪水、社會壓力和同儕影響，甚至是為了逃避社會壓力
無論是出於哪個動機，重要的是在申請研究所前仔細思考，確保自己的決定是基於充分的考慮和個人目標。研究所生涯需要投入大量時間和精力，應該確保自己對此有充分的準備和動力。

**Comment by jim1743** [comment]
我自己也是偏向二或三，畢竟家裡的經濟環境也不是到很富有，未來工作的薪資就變得很重要，再來現在就業環境這麼激烈，我又是理工背景的，如果不跟上其他人選擇讀研究所，那我在未來選擇應徵的公司又有甚麼理由選擇我當他的員工。

**Comment by EPHMR0524** [comment]
我也是偏向2或是3，因為現在碩士真的是滿街跑，隨便路上撞到一個人可能都是碩士，且理組人沒讀個碩士感覺就不會有公司願意收。
現在社會競爭真是相當激烈，我覺得還是讀個碩士當保障會比較好，此外，在碩班就是花大部分的心力在研究上面，不像大學部的課程內容一樣多數會有正確答案，在這樣的過程中我覺得能學到更多東西。

**Comment by YuanWang08** [comment]
在台灣，人人有錢都有大學念，大學文憑氾濫，導致平均價值下降，研究所很重要似乎是台灣特有的現象，在國外，就算是科技大廠，使用大學學歷的員工也不在少數，我的意思是與台灣相比。當然有碩士學位會增加你的客觀評價，在台灣應該可以說是不可或缺了，甚至有名的公式都要求至少要有碩士才有面試資格，這在國外是比較少見的。

**Comment by yxin214** [comment]
讀研究所對我來說比較像是，因為身邊同學的規劃都是繼續考研，我也很自然地認為這會出現在我的規劃中。我是覺得讀研究所能讓我的專業學習的更好，未來找工作也更有利，是有必要的。

**Comment by jylin34** [comment]
對我來說，讀研究所，提高工作薪水勢必是一個因素，但更重要的是能體驗過、學習作research的過程，從一開始發想主題、查資料佐證、如何提案說服其他人的想法、最後發表成果，我認為這個能力對未來職場也是十分重要的。有遇過業界的人士，研究所學的領域其實都跟工作內容相差甚遠，但是有research能力，到什麼領域都會受用。

**Comment by alexender0104** [comment]
對於我來說，我認為薪水是最重要的，比較大學畢業的薪資與研究所畢業的薪水，中間有一段落差，cp質差蠻多的，以及讀研究所等於多了兩年的學生時間，雖然有些實驗室很血汗，但比起社會，學校還是稍微單純點。

**Comment by OHoln** [comment]
我認為主要還是看科系吧，有些科系有沒有讀研起薪就是差挺多的，而且第一份工作找好點對接下來的職業生涯也很有幫助，但如果對就業沒有太大影響我認為就是個人興趣的選擇了。

**Comment by SusuTima** [comment]
曾經聽過一個說法，讀完大學就跟讀完幼稚園一樣，這種說法是稍微有一點言之過重了，但是大學時期也確實都還在打基礎，對於專精的領域無法有完整的時間去鑽研，所以讀研究所，某方面程度也是在專精自己的研究領域。當然啦，每個人看待研究所的角度不同，我自己的觀點也偏向2.3.，畢竟別人有這份學歷，我沒有，感覺輸在起跑點上了。

**Comment by hbh10601** [comment]
應該說碩士學位的天花板會比學士學位來的高，主要還是看個人未來想要的發展程度

**Comment by Shang621** [comment]
研究所的必要性因人而異。對於那些渴望深入研究、追求學術成就的人，研究所提供了專業知識和研究能力的提升平台，並且有助於建立有價值的人脈。另一方面，考慮到高昂的學費和時間成本，並非所有人都需要進入研究所。有些人通過職場經驗和自學，同樣能夠獲得成功。因此，選擇是否進入研究所，應該根據個人目標和需求來決定，而非盲目跟風。

**Comment by tshsu04224** [comment]
個人是認為研究所對未來就業的薪水確實是有一定程度的影響，不升研究所直接出去工作，怕會面臨到薪資天花板的問題，因此覺得唸研究所好像是對未來比較正確的選擇，多花一些時間投資未來吧

**Comment by sandyho0307** [comment]
我覺得真心喜愛這門學科或研究領域是非常重要的。畢竟，只有對某個領域充滿熱情和興趣，才能在長期的學習和研究中保持動力。這樣的熱情不僅會讓我們在研究所的學習更加投入，也會在未來的職業生涯中帶來更大的滿足感。
此外，從現實的角度來看，薪水和職業前景也是考慮的重點。據我了解，碩士學位通常可以帶來更高的起薪和更多的職業機會。對於我們商管科系的學生來說，高學歷在進入一些高薪職位或者晉升管理層時，確實有一定的優勢。因此，為了提升自身的競爭力，讀研究所是一個不錯的選擇。

**Comment by asd99263fg** [comment]
學歷只是一張紙而已，很多時候不是看你到底學到了什麼，而是你到底在這個過程得到了什麼，的確你還是要有個文憑畢業，但是有了學歷不代表人生就一帆風順，還是有很多挑戰等著你完成，所以繼續努力把

**Comment by cassy0201** [comment]
我發現目前社會給研究所畢業的底線確實比大學畢業的還高，但相對也比別人少賺兩年的薪水與經驗，之前聽學長姐分享,他們認為長遠來看是差不多的,這也讓我開始考慮是否要踏入又研究所了

**Comment by 155Julia** [comment]
我主要是選擇二，因為有沒有研究所真的差蠻多的。對於我來說，進入研究所不僅能夠提高我的專業知識，還能在職場上獲得更高的薪資和更好的職涯發展。我認為碩士學位往往代表著更高的專業能力和更多的工作機會。

**Comment by yubo0oo** [comment]
我是在乎薪水
碩士跟學士的起薪和漲薪幅度也差距不少

**Comment by asd99263fg** [comment]
專業知識的深化：研究所課程通常專注於特定領域，讓學生能夠深入了解和掌握該領域的專業知識和技能。
職業發展：擁有研究所學位可以增加職場競爭力，許多高級職位和專業職業要求碩士或博士學位。此外，研究所畢業生通常有更高的薪資起點和更好的職業發展機會。
研究能力的培養：研究所課程通常包括大量的研究項目，學生有機會在指導教授的指導下進行獨立研究，這有助於培養解決問題的能力和批判性思維。
學術交流和人脈建立：在研究所學習期間，學生有機會與來自世界各地的學者和專家交流，參加各類學術會議和研討會，這有助於建立專業人脈和拓展視野。
個人興趣的探索：研究所提供了深入探索自己感興趣的學術領域的機會，讓學生可以追求自己的學術興趣和研究目標。
學術成就感：完成研究所學位，尤其是博士學位，是一項重要的學術成就，能夠帶來極大的自豪感和成就感。
教學和研究職位：研究所學位是進入學術界的重要門檻，許多教學和研究職位都要求擁有碩士或博士學位。

創新和貢獻：在研究所學習期間，學生有機會進行原創研究，為學術界和實踐領域做出貢獻，推動相關領域的發展和創新。
